integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -611,20 +612,37 @@ def test_radians(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : unary_op_df(spark, data_gen).selectExpr('radians(a)'))
 
+def get_java_version():
```

Comment:
nit: Would it be better to use the java bridge

```
spark._jvm.java.lang.Runtime.version()
```

And if that works should we move this to a common place, like with `is_before_spark_320()`...

I don't see a lot of others wanting to use this, but it would be nice to have it in a common place anyways.

Reply:
Yes, java bridge is better, thanks. I used `System.getProperty("java.version")` because `Runtime.version()` is since JDK9.